README.md

```diff
@@ -2,0 +1,17 @@
+# codacy-test
+
+### Among Codacy’s features
+- Identify new Static Analysis issues
+* Commit and Pull Request Analysis with GitHub, BitBucket/Stash, GitLab (and also direct git repositories)
+- Auto-comments on Commits and Pull Requests
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://release.k8s.dev.codacy.org/gh/DevK8scodacytestuser/codacy-test/pullRequest?prid=3215)
```suggestion

- Auto-comments on Commits and Pull Requests
```